HPXMLtoOpenStudio/resources/output.rb

```diff
@@ -203,10 +204,20 @@ def self.apply_total_loads_ems_program(model, hpxml_osm_map, hpxml_header)
       end
 
       # Energy transferred in conditioned zone, used for determining heating (winter) vs cooling (summer)
+      # Add supp heat delivered energy to the meter, using Meter:Custom
+      htg_load_key_vars = [['', "Heating:EnergyTransfer:Zone:#{conditioned_zone_name.upcase}"]]
+      model.getOtherEquipments.sort.each do |o|
+        next unless o.endUseSubcategory.start_with? Constants::ObjectTypeHPDefrostSupplHeat
+        next unless o.space.get.thermalZone.get.name.to_s.upcase == conditioned_zone_name.upcase
+
+        htg_load_key_vars << [o.name.to_s.upcase, 'Other Equipment Total Heating Energy']
```

Comment:
This doesn't seem working because the object level output variable doesn't incorporate thermal zone multiplier, need to revert this approach and try using sensors in the total_load EMS

Reply:
@yzhou601 There should be an energy meter that corresponds to each Equipment object (that would then incorporate the thermal zone multiplier). See [here](https://github.com/NREL/OpenStudio-HPXML/blob/master/ReportSimulationOutput/measure.rb#L3121-L3152) for example.

Reply:
In particular, [this part](https://github.com/NREL/OpenStudio-HPXML/blob/master/ReportSimulationOutput/measure.rb#L3145-L3151).